src/content/resources/faq.md

```diff
@@ -715,17 +729,21 @@ if Dart can identify the code path at compile time.
 To date, we've found other approaches for specific needs
 that offer a better trade-off, such as code generation.
 
-### How do I do international&shy;ization (i18n), localization (l10n), and accessibility (a11y) in Flutter?
+### Is internationalization and localization supported?
 
-Learn more about i18n and l10n in the
-[internationalization tutorial][].
+Flutter supports internationalization (i18n) and localization (l10n) so that
```

Comment:
Not sure if this is intentional, but I feel like we should start with a yes or no answer if it's a a yes/no question?

```suggestion
Yes, Flutter supports internationalization (i18n) and localization (l10n) so that
```